auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -382,12 +475,29 @@ export default class AccountInfo extends Mixins(AccountChangeMixin) {
     return false
   }
 
+  get isAccountStatusActive () : boolean {
+    return this.currentOrganization.statusCode === AccountStatus.ACTIVE
```

Comment:
Again ; the logic is show UNSUSPEND if status is SUSPENDED..  for all other status , we need to show suspend..we have other statuses like PENDING_AFFIDAVIT_REVIEW for which we should show suspend button